### PR TITLE
Management UI update

### PR DESCRIPTION
**Pull request by loneil**
# Description
https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-1640

Also 
https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-1641
https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-1642

New management UI and feature flag restrictions (must be CHEFS admin for some stuff).
Delete button not functional yet (hidden anyways).

Will add version history table at bottom at a later time with design changes.

## Types of changes

New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-wxpbtr-tools.pathfinder.gov.bc.ca/dashboard?id=common-hosted-form-service-pr-50

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-wxpbtr-tools.pathfinder.gov.bc.ca/dashboard?id=common-hosted-form-service-pr-50

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-wxpbtr-tools.pathfinder.gov.bc.ca/dashboard?id=common-hosted-form-service-pr-50

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-wxpbtr-tools.pathfinder.gov.bc.ca/dashboard?id=common-hosted-form-service-pr-50

File: app/frontend/src/views/form/Manage.vue [hunk 1]
```diff
@@ -1,141 +1,232 @@
 <template>
   <div>
-    <v-breadcrumbs :items="breadcrumbs" />
```
Comment (parc-jason) [inline app/frontend/src/views/form/Manage.vue:3]: Should we be getting rid of all breadcrumbs?

File: app/frontend/src/views/form/Manage.vue [hunk 2]
```diff
@@ -1,141 +1,232 @@
...
-              <span>Modify</span>
-            </v-btn>
-          </v-card-text>
-        </v-card>
-      </v-col>
-    </v-row>
+          </p>
+
+          <BaseInfoCard>
+            Editing this form and saving the changes will create and publish a
```
Comment (parc-jason) [inline app/frontend/src/views/form/Manage.vue:127]: May need to run this by Mina, but we should keep differentiating between Form (Form Settings) and Form Design.  So this should probably say 'Editing this form design'

File: app/frontend/src/views/form/Manage.vue [hunk 3]
```diff
@@ -1,141 +1,232 @@
...
               :to="{
                 name: 'FormDesigner',
                 query: { f: f, v: currentVersion.id },
               }"
             >
-              <v-btn color="blue" text small>
-                <v-icon class="mr-1">edit</v-icon>
+              <v-btn v-if="canCreateDesign" color="primary" text small>
+                <v-icon class="mr-1 ml-5">edit</v-icon>
                 <span>Edit Current Form</span>
```
Comment (parc-jason) [inline app/frontend/src/views/form/Manage.vue:121]: See below, perhaps run by Mina about saying 'Edit Current (Form) Design'.

**Review by parc-jason — APPROVED** [review]
Looks good, huge improvement... ever closer.
I have questions, but I know we are doing more rounds of QA and tweaks and those can all be raised at that time.

File: app/frontend/src/views/form/Manage.vue (hunk 1 above)
Comment (loneil) [reply app/frontend/src/views/form/Manage.vue:3]: Yeah there's a ticket to remove them all for initial release. SHOWCASE-1639

File: app/frontend/src/views/form/Manage.vue (hunk 2 above)
Comment (loneil) [reply app/frontend/src/views/form/Manage.vue:127]: That's a good point. This is temp wording so didn't get it from the mockups. Will change to that.

File: app/frontend/src/views/form/Manage.vue (hunk 1 above)
Comment (parc-jason) [reply app/frontend/src/views/form/Manage.vue:3]: should've known.  cool.

**Pull request merged**